stubs/_nimblephysics-stubs/dynamics/__init__.pyi

```diff
@@ -4493,6 +4501,22 @@ class Skeleton(MetaSkeleton):
     def enableSelfCollisionCheck(self) -> None: ...
     def fitJointsToWorldPositions(self, positionJoints: typing.List[Joint], targetPositions: numpy.ndarray[numpy.float64, _Shape[m, 1]], scaleBodies: bool = False, convergenceThreshold: float = 1e-07, maxStepCount: int = 100, leastSquaresDamping: float = 0.01, lineSearch: bool = True, logOutput: bool = False) -> float: ...
     def fitMarkersToWorldPositions(self, markers: typing.List[typing.Tuple[BodyNode, numpy.ndarray[numpy.float64, _Shape[3, 1]]]], targetPositions: numpy.ndarray[numpy.float64, _Shape[m, 1]], markerWeights: numpy.ndarray[numpy.float64, _Shape[m, 1]], scaleBodies: bool = False, convergenceThreshold: float = 1e-07, maxStepCount: int = 100, leastSquaresDamping: float = 0.01, lineSearch: bool = True, logOutput: bool = False) -> float: ...
+    def getAccMapReadings(self, accelerometers: typing.Dict[str, typing.Tuple[BodyNode, nimblephysics_libs._nimblephysics.math.Isometry3]]) -> typing.Dict[str, numpy.ndarray[numpy.float64, _Shape[3, 1]]]: 
+        """
+        These are a set of bodies, and offsets in local body space where gyros
+        are mounted on the body
+            
+        """
+    def getAccelerometerReadings(self, accelerometers: typing.List[typing.Tuple[BodyNode, nimblephysics_libs._nimblephysics.math.Isometry3]]) -> numpy.ndarray[numpy.float64, _Shape[m, 1]]: 
```

Comment:
In getGyroReadings, "gyros" should be formatted as follows, "typing.Dict[str, typing.Tuple[BodyNode, nimblephysics_libs._nimblephysics.math.Isometry3]]". Here "accelerometers" is a list.